scripts/host-only/wkdev-sdk-bakery

```diff
@@ -58,7 +59,16 @@ build_image() {
 
     pushd "$(get_image_directory_by_name "${container_image_name}")" &>/dev/null
     timer_start
-    run_podman_silent_unless_verbose build --jobs $(get_number_of_cores_for_build) --tag "$(get_tag_for_build)" . || _abort_ "Container image build failed"
+
+    local podman_argument=("--jobs" "$(get_number_of_cores_for_build)")
+    podman_argument+=("--tag" "$(get_tag_for_build)")
+
+    for environment_variable in "${cumulated_values_env[@]}"
+    do
```

Comment:
```suggestion
    for environment_variable in "${cumulated_values_env[@]}"; do
```

Reply:
Missed that suggestion, sorry -- will fix it up later.